neofetch

```diff
@@ -2178,6 +2178,32 @@ END
                          "${XDG_CONFIG_HOME}/sakura/sakura.conf")"
         ;;
 
+        "st")
+            if term_font="$(ps -o command= -p "$parent" | grep -E '\-f\s?.*?:?')"; then
+                term_font="${term_font/*-f/}"
+                term_font="${term_font/ -*/}"
+            else
+                binary=""
```

Comment:
This isn't needed.

Reply:
From what I can see this is still needed:

```bash
                term_font="${term_font/*-f/}"
                term_font="${term_font/ -*/}"
```

I've removed the `binary=""`